docs/tutorials/demo/Alchemiscale Demo.ipynb

```diff
@@ -0,0 +1,2774 @@
+      "lig_ejm_47 lig_ejm_31\n",
+      "lig_ejm_46 lig_ejm_31\n"
+     ]
+    }
+   ],
+   "source": [
+    "for mapping in ligand_network.edges:\n",
+    "    print(mapping.componentA.name, mapping.componentB.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "f3379b4a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "complexed_transformations = []\n",
+    "solvated_transformations = []\n",
+    "\n",
+    "for mapping in ligand_network.edges:\n",
+    "    ligA_name = mapping.componentA.name\n",
+    "    ligB_name = mapping.componentB.name\n",
+    "    \n",
+    "    ligA = complexed[ligA_name]['ligand']\n",
+    "    ligB = complexed[ligB_name]['ligand']\n",
+    "        \n",
+    "    complexed_transformations.append(\n",
+    "        openfe.Transformation(stateA=complexed[ligA_name], \n",
+    "                              stateB=complexed[ligB_name], \n",
+    "                              mapping={'ligand': mapping},\n",
+    "                              protocol=protocol,\n",
+    "                              name=f\"{complexed[ligA_name].name}->{complexed[ligB_name].name}\")\n",
+    "    )\n",
+    "    solvated_transformations.append(\n",
+    "        openfe.Transformation(stateA=solvated[ligA_name], \n",
+    "                              stateB=solvated[ligB_name], \n",
+    "                              mapping={'ligand': mapping},\n",
+    "                              protocol=protocol,\n",
+    "                              name=f\"{solvated[ligA_name].name}->{solvated[ligB_name].name}\")\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3789edec-beaf-4bae-903b-da35910f848e",
+   "metadata": {},
+   "source": [
+    "### Create the `AlchemicalNetwork`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "618d399e-d66a-499e-b43a-684b5e1b8ec5",
+   "metadata": {},
+   "source": [
+    "An `AlchemicalNetwork` is simply the combination of `ChemicalSystem`s (nodes) and `Transformation`s (directed edges) that we want to evaluate $\\Delta G$s for. This data structure functions as a declaration of what you want to compute.\n",
+    "\n",
+    "We'll finish here by creating an `AlchemicalNetwork` from the collection of objects we've built so far."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "b45baada-efa4-46f7-a2cb-6e24a9309f1e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<AlchemicalNetwork-391c0eb68025cf4b83e4d706f189f745>"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "network = AlchemicalNetwork(edges=(solvated_transformations + complexed_transformations), \n",
+    "                            nodes=(list(solvated.values()) + list(complexed.values())),\n",
+    "                            name=\"tyk2_relative_benchmark\")\n",
+    "network"
```

Comment:
We've improved the network representtation post openfe 0.11, it should look a lot less cluttered

Reply:
How do you mean? This is just defining the network itself; do you mean some kind of visualization?